gatsby-config.esm.js

```diff
@@ -46,6 +53,10 @@ module.exports = {
         excludes: ['/help', '/contact'],
       },
     },
+    `gatsby-transformer-json`,
+    'gatsby-plugin-image',
+    'gatsby-plugin-sharp',
```

Comment:
there's a lot of image optimization that i didn't do that plugin-sharp can do.  Mostly just transforming the images to match the pixels they'll use at build time